dockerfile

```
@@ -0,0 +1,21 @@
+FROM ubuntu:20.04
+
+RUN useradd -M pgbouncer && \
+    apt-get -y update && \
+    apt-get -y install pgbouncer && \
+    rm -rf /var/lib/apt/lists/* && \
+    chown -R pgbouncer /etc/pgbouncer
+
+USER pgbouncer
+
+WORKDIR /etc/pgbouncer
+
+COPY --chown=pgbouncer ./start_pgbouncer.sh start_pgbouncer.sh
```

By setting the `WORKDIR` above, the result of this line is that you end up with `/etc/pgbouncer/start_pgbouncer.sh`, I think you should have this script at `/entrypoint.sh` just for the sake of convention -- startup scripts don't belong in `/etc` :)
```suggestion
COPY --chown=pgbouncer ./start_pgbouncer.sh /entrypoint.sh
```